### ISSUE TITLE
test_ae_classification_network_converges gives accuracy > 1

### ISSUE THREAD

**Issue by MaddyThakker** [traja-team/traja#57]
File `test_models.py` implements `test_ae_classification_network_converges` (line 334) which during validation `trainer.validate(data_loaders['train_loader'])` (line 406) gives accuracy > 1.

**Comment by MaddyThakker** [comment]
The reason for this is `predicted` is Tensor of shape `torch.Size([10])` but `classes` is a Tensor of shape `torch.Size([10, 1])`. Need to reshape the classes to same size as of predicted to make it work.

**Comment by MaddyThakker** [comment]
Resolved with https://github.com/traja-team/traja/pull/66
